index.js

```diff
@@ -37,34 +69,31 @@ class Main {
     });
   }
 
-  async disconnect() {
-    // This call should be refactored to work with async/await
-    this.exporter.disconnect();
-    await this.microServer.close();
+  _updateMetrics() {
+    metrics.currentBlock.set(this.worker.lastConfirmedBlock);
+    metrics.requestsCounter.inc(this.worker.getNewRequestsCount());
+    metrics.requestsResponseTime.observe(new Date() - this.worker.lastRequestStartTime);
```

Comment:
we can do subtraction in grafana. but we loose info if we do it here and send deltas instead of absolute value